packages/vue/src/components/atoms/SfColor/SfColor.stories.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from "@storybook/vue";
+import { withKnobs, optionsKnob as options } from "@storybook/addon-knobs";
+
+import SfColor from "./SfColor.vue";
+
+storiesOf("Atoms|Color", module)
+  .addDecorator(withKnobs)
+  .add("Common", () => ({
+    components: { SfColor },
+    data() {
+      return {
+        colors: [
+          {
+            label: "blue",
+            active: false
+          },
+          {
+            label: "yellow",
+            active: false
+          },
+          {
+            label: "pink",
+            active: true
+          },
+          {
+            label: "orange",
+            active: false
+          },
+          {
+            label: "white",
+            active: false
+          }
+        ]
+      };
+    },
+    props: {
+        customClass: {
+          default: options(
+            "CSS modifiers",
+            {
+              "sf-color--light": "sf-color--light",
```

Comment:
What is the point of this modifier?

Reply:
Its for almost white/light colors, so it won't be invisible

Reply:
Maybe  rename the modifier to `--border`as it says more clearly what will happen

Reply:
Done